dash/include/dash/algorithm/Accumulate.h

```diff
@@ -86,45 +165,23 @@ ValueType accumulate(
 template <
   class GlobInputIt,
   class ValueType,
-  class BinaryOperation >
+  class BinaryOperation = dash::plus<ValueType>,
```

Comment:
Please use our iterator traits to check if `GlobInputIt` is really a global iterator. This is more readable.